### DISCUSSION TITLE
Brew services

### DISCUSSION THREAD

**Discussion by hamzaelmaghari** [beyondcode/herd-community#334] [Q&A] [answered]
Hi,
I just wanna know if Herd has the ability to add brew services in the menu?

**Comment by mpociot** [comment, marked as answer]
No, Herd does not manage any brew services

**Reply by hamzaelmaghari** [reply]
Thanks, it's a great feature I hope Herd include it.
For me I used to use [Phpmon](https://github.com/nicoverbruggen/phpmon) which allows me to manage Homebrew services, then I switched to Herd due to it's features.

Anyway thanks,

**Reply by bashgeek** [reply]
One of the features of Herd is, that you do NOT need Brew anymore for anything but have fast, static builds of everything included, so doubt that'll happen.
